srv/modules/runners/validate.py

```diff
@@ -107,6 +107,20 @@ def _clusters(self):
             clusters.setdefault(cluster, []).append(minion)
         return clusters
 
+
+class Util(object):
+    @staticmethod
+    def parse_list_from_string(list_str, delim=','):
```

Comment:
why not a list comprehension or something little more simpler there,,
```py
[x for x in xs.split(",") if x != '']  
``` 
for eg